ONSdigital/rm-case-service
# RAS-939 Survey Status Change

# What and why?
Adds a new case group transition from `complete` to `not started` to allow business users a new status change in rOps
# How to test?
- deploy this and https://github.com/ONSdigital/response-operations-ui/pull/914 to your environment
- run the acceptance tests to populate your env
- log into frontstage and complete the survey
- go to http://localhost:8085/reporting-units/49900000001 and navigate to change the status from complete to not started
- check that the survey appears in the todo page in frontstage
# Jira
https://jira.ons.gov.uk/browse/RAS-939
